jenkins.sh

```diff
@@ -6,4 +6,9 @@ rm -f Gemfile.lock
 git clean -fdx
 bundle install --path "${HOME}/bundles/${JOB_NAME}"
 
-bundle exec rake spec build:and_release_if_updated
+bundle exec rake spec
+
+# If $TRAVIS is unset, try and release the template
+if [ -z "$TRAVIS" ]; then
+  bundle exec rake build:and_release_if_updated
+fi
```

Comment:
It would be better to make this only release if a variable is set - safer that way.  Then we can set the variable for the master Jenkins job.

This is how other gems that do branch builds work - eg: https://github.com/alphagov/gds-api-adapters/blob/master/jenkins.sh